test/e2e/azure_logcollector.go

```diff
@@ -125,6 +125,11 @@ func (k AzureLogCollector) CollectMachinePoolLog(ctx context.Context, management
 	return kinderrors.NewAggregate(errs)
 }
 
+// CollectInfrastructureLogs collects log from the infrastructure.
+func (k AzureLogCollector) CollectInfrastructureLogs(ctx context.Context, managementClusterClient client.Client, c *clusterv1.Cluster, outputPath string) error {
```

Comment:
This is a dummy implementation of a new func necessary to satisfy the log collector interface.

Reply:
Just to clarify, does that mean there are logs being thrown out here? Or would it just be duplicates of logs we're already collecting our own way?

Reply:
nit: I think it could help to specify that this is a no-op to satisfy the interface as a comment

Reply:
Done.